src/main/java/seedu/address/model/export/DocumentPath.java

```diff
@@ -100,11 +105,23 @@ public String toString() {
         return directoryPath.toString() + File.separator + documentFilePath.toString();
     }
 
+    @Override
+    public void export(List<FlashCard> list) throws IOException {
+        try {
+            DocumentExportUtil.exportFlashCardsToDocument(
+                    list, this
+            );
+        } catch (IOException e) {
+            throw e;
+        }
```

Comment:
No need for a try/catch since method signature already `throws IOException`

Reply:
You're right in saying that it's redundant, but I feel that this bit of redundancy leads to a better programming practice. Here's an excerpt from slide 21 of CS2030 Lecture 4 notes (under Prof Zhao Shendong):
![image](https://user-images.githubusercontent.com/41581512/66898099-03d40f80-f02b-11e9-906b-4fc94a069508.png)